### ISSUE TITLE
Can we use auc or accuracy rather than logloss?

### ISSUE THREAD

**Issue by jinxin0924** [zygmuntz/hyperband#1]
`loss = result['loss']`
`val_losses.append( loss )`

Can we replace the loss with auc if I am more interested in auc? Or logloss will be better even if I care about auc?

**Comment by zygmuntz** [comment]
Yes We Can! See https://github.com/zygmuntz/hyperband/blob/master/common_defs.py#L69

Since we're minimizing the loss, remember to use 1 - auc
